docker/build.sh

```diff
@@ -71,7 +73,10 @@ docker_version_check() {
 build_and_push() {
   docker_version_check
 
-  docker buildx create --name img-builder --use
+	# only moby/buildkit:foreign-mediatype works on building Windows image now
+	# https://github.com/moby/buildkit/pull/1879
+	# Github issue: https://github.com/moby/buildkit/issues/1877
```

Comment:
nit: i think this is using tabs when rest of file is spaces

Reply:
thank you. It should be updated now.